Sources/DeepLearning/Layer.swift

```diff
@@ -1169,6 +1169,35 @@ public struct UpSampling2D<Scalar: TensorFlowFloatingPoint>: Layer {
     }
 }
 
+/// An upsampling layer for 3-D inputs.
+@_fixed_layout
+public struct UpSampling3D<Scalar: TensorFlowFloatingPoint>: Layer {
+    @noDerivative public let size: Int
+
+    /// Creates an upsampling layer.
+    ///
+    /// - Parameter size: The upsampling factor for rows and columns.
+    public init(size: Int) {
+       self.size = size
+    }
+
+    /// Returns the output obtained from applying the layer to the given input.
+    ///
+    /// - Parameter input: The input to the layer.
+    /// - Returns: The output.
+    @differentiable
+    public func call(_ input: Tensor<Scalar>) -> Tensor<Scalar> {
+        let shape = input.shape
+        let (batchSize, height, width, depth, channels) =
+                (shape[0], shape[1], shape[2], shape[3], shape[4])
```

Comment:
Could you please make sure they are indented by 4 spaces from `let`?
```suggestion
            (shape[0], shape[1], shape[2], shape[3], shape[4])
```

Reply:
Oh I made them 4 tabs instead of spaces. My bad.